callback_plugins/default_plus_summary.py

```diff
@@ -0,0 +1,94 @@
+# vim: expandtab:tabstop=4:shiftwidth=4
+'''
+Ansible callback plugin.
+'''
+
+from ansible.plugins.callback.default import CallbackModule as CallbackModule_default
+from ansible import constants as C
+from ansible.utils.color import stringc
+
+
+class CallbackModule(CallbackModule_default):
+    '''
+    This is like the default callback plugin, but also stores results and
+    summarizes failures.
+    '''
+
+    CALLBACK_VERSION = 2.0
+    CALLBACK_TYPE = 'stdout'
```

Comment:
If set to aggregate would the normal stdout plugin still run?  Would the custom one simply run afterwards?  The idea of this callback plugin was to have reduced stdout that includes better error message aggregation.